dist/ramda.js

```diff
@@ -6200,10 +6198,6 @@
     };
 
     var R = {
-        F: F,
-        I: I,
-        T: T,
-        __: __,
```

Comment:
Does this somehow explain the build failure?  I don't understand _why_ `make` failed.